jetty-ee10/jetty-ee10-servlet/src/main/java/org/eclipse/jetty/ee10/servlet/ServletHandler.java

```diff
@@ -491,6 +491,18 @@ public MatchedResource<MappedServlet> getMatchedServlet(String target)
         return new MatchedResource<>(holder, null, MatchedPath.EMPTY);
     }
 
+    /**
+     * ServletHolder for the default servlet.
+     *
+     * @return MatchedResource, pointing to the {@link MappedResource} for the {@link ServletHolder}.
+     */
+    public MappedResource<MappedServlet> getDefaultServlet()
```

Comment:
How is this different from `getServletMapping("/")`

Reply:
This will give you the server mapped to the context root, which may not be the default servlet.